src/main/routes/isAlive.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * A quick check to verify that there is an HTTP server available, indicating
+ * that the app is "up" regardless of the current state of any external
+ * dependencies.
+ */
+import {Application, Request, Response} from 'express';
+
+export default (app: Application): void => {
+    app.get('/isAlive', (req: Request, res: Response) => res.json({isAlive: "yes"}));
```

Comment:
A deliberately simple, instant, "is there a server process alive?" healthcheck - The existing one is too weighty and contains too many dependencies which could lead to churn in the ECS tasks